app/src/components/map.jsx

```diff
@@ -211,28 +217,6 @@ class Map extends Component {
   }
 
   /**
-   * Draws a line that connects by timestamp order all paths of the following vessel
```

Comment:
Are you sure we want to remove that bit?
Before that I was able to click on vessels and see the red path.